PyPoE/cli/exporter/wiki/parser.py

```diff
@@ -1537,20 +1540,16 @@ def _format_detailed(self, custom, ingame):
 
     def _write_dds(self, data, out_path, parsed_args):
         out_path = fix_path(out_path.replace('"', "'", 2))
-        with open(out_path, 'wb') as f:
-            f.write(self.file_system.extract_dds(data))
-
-            console('Wrote "%s"' % out_path)
+        if parsed_args.convert_images:
+            out_img = decode_dds(data)
+            out_img.save(out_path.replace('.dds', '.png'))
 
-        if not parsed_args.convert_images:
-            return
-
-        os.system('process-image convert "%s" "%s"' % (
-            out_path, out_path.replace('.dds', '.png'),
-        ))
-        os.remove(out_path)
+            console('Converted "%s" to png' % out_path)
+        else:
+            with open(out_path, 'wb') as f:
+                f.write(self.file_system.extract_dds(data))
```

Comment:
You could import `Path` from `pathlib` to make use of `Path`'s built-in context manager to avoid this.
Instead the whole thing could be written as:

```py
Path(out_path).write_text | .write_bytes (self.file_system.extract_dds(data))
```